leaderLogs/leaderLogs.py

```diff
@@ -7,6 +7,29 @@
 import hashlib
 from ctypes import *
 
+import json
+import argparse
+
+parser = argparse.ArgumentParser(description="Calculate the leadership log.")
+parser.add_argument('--pool-id', dest='poolId', help='the pool ID', required=True)
+parser.add_argument('--epoch', dest='epoch', help='the epoch number [e.g. 221]', type=int, required=True)
+parser.add_argument('--epoch-nonce', dest='eta0', help='the epoch nonce to check', required=True)
+parser.add_argument('--vrf-skey', dest='skey', help='provide the path to the pool.vrf.skey file', required=True)
+parser.add_argument('--sigma', dest='sigma', type=float, help='the controlled stake sigma value of the pool', required=True)
```

Comment:
maybe give an example.  some people might submit 1% or something.  0.010052348379780869
`the controlled stake sigma value of the pool (example: 0.0034052348379780869) `